crates/net/network/src/manager.rs

```diff
@@ -186,7 +184,7 @@ where
             dns_discovery_config,
             extra_protocols,
             tx_gossip_disabled,
-            transactions_manager_config,
+            transactions_manager_config: _,
```

Comment:
please remove

Reply:
but currently transactions_manager_config is a field in networkconfig, i don't know any way to remove it as mentioned above. https://github.com/paradigmxyz/reth/pull/6594#discussion_r1490157325

Reply:
the `: _` should be removed, not the entire line

Reply:
no the entire line, it's not being stored in network manager :)